client/my-sites/themes/theme-showcase.scss

```diff
@@ -103,6 +102,15 @@ body.is-section-themes {
 }
 
 body.is-section-themes-i4-2 {
```

Comment:
Could you explain the `-2` in the body class name, like to indicate it's phase 2 of the project?

Reply:
Yep, this is so that the CSS updates is applied together with the changes under the flag `themes/showcase-i4/details-and-preview`.

Reply:
Thanks